insights/specs/datasources/sys_fs_cgroup_memory.py

```diff
@@ -0,0 +1,44 @@
+"""
+Custom datasources for the files under /sys/fs/cgroup/memory
+"""
+import os
+
+from insights.core.context import HostContext
+from insights.core.dr import SkipComponent
+from insights.core.plugins import datasource
+from insights.core.spec_factory import DatasourceProvider
+
+
+@datasource(HostContext)
+def sys_fs_cgroup_uniq_memory_swappiness(broker):
+    """
+    This datasource reads all the memory.swappiness files under /sys/fs/cgroup/memory directory
+    one by one and returns the uniq memory swappiness setting of the all control groups.
+
+    The output of this datasource looks like:
+        10   1
+        60   66
+
+    Returns:
+        str: Returns a multiline string in the format as ``value  count``.
```

Comment:
nitpick: I saw the number of spaces between the `value` and `count` here is different from others.